rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -835,16 +835,17 @@ def _set_wait(self, request, validator_query):
 
     @staticmethod
     def _format_statuses(statuses):
-        """Reformat converted BatchStatus dicts: drop empty keys, rename 'id'.
+        """Reformat converted BatchStatus dicts: rename keys and drop empties.
         """
         for status in statuses:
             status['id'] = status.pop('batch_id')
+            status['invalid_message'] = status.pop('message', '')
```

Comment:
Why rename these?  It makes it so the message is only for invalid statuses, and can't be used for any other status.  No reason that valid transactions could return extended data in the future.